mantidproject/mantid
# Indirect Bayes - Add plot Current Preview

Added functionality for creating a plot of the current miniplot for Indirect>Bayes interfaces.

**To test:**

Input Files:
[ResNorm.zip](https://github.com/mantidproject/mantid/files/827784/ResNorm.zip)
[Quasi - Stretch.zip](https://github.com/mantidproject/mantid/files/827783/Quasi.-.Stretch.zip)

Interfaces>Indirect>Bayes

For each interface check that the "current spectrum" spin box changes the miniplot.
Click "Plot Current Preview" button and ensure that plot produced matches the current spectrum on the miniplot.
Try plotting both before and after pressing `Run`

ResNorm:
Vanadium: `iris26173_graphite002_red`
Resolution: `iris26173_graphite002_res`

Quasi & Stretch:
Sample: `iris26176_graphite002_red`
Resolution: `iris26173_graphite002_res`

Note that Stretch doesn't actually fit anything so the miniplot should only display the sample data.

Fixes #17262.

Release Notes have been updated

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
